modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-alert-routing-for-user-defined-projects_{context}"]
+= Creating alert routing for user-defined projects
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Alert routing for user-defined projects is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+[role="_abstract"]
+You can create alerting routing for user-defined projects. 
+
+.Prerequisites
+
+* An administrator has enabled monitoring for user-defined projects.
```

Comment:
and also enabled alerting routing for user-defined projects

Reply:
That's a good point. When flowing through the docs, the current format actually makes sense since you enable alerting -> (next steps) create alerts but there is no harm in calling it out or even linking back here again explicitly

Reply:
Added item to the bulleted list